lib/mayaUsd/ufe/UsdAttributeHolder.h

```diff
@@ -29,10 +29,8 @@ namespace ufe {
 //! \brief Internal helper class holding a USD attributes for query:
 class UsdAttributeHolder
 {
-protected:
-    UsdAttributeHolder(const PXR_NS::UsdAttribute& usdAttr);
-
 public:
+    UsdAttributeHolder(const PXR_NS::UsdAttribute& usdAttr);
```

Comment:
Allow creating a quick transient holder to access enum information.

Reply:
Another alternative would be to make my suggested function be static and pass the UsdAttribute to it and be public.